test/lib/y2firewall/clients/proposal_test.rb

```diff
@@ -24,14 +24,19 @@
 require "y2firewall/clients/proposal"
 
 describe Y2Firewall::Clients::Proposal do
-  let(:client) { described_class.new }
+  subject(:client) { described_class.new }
   let(:proposal_settings) { Y2Firewall::ProposalSettings.instance }
 
+  before do
+    # skip bootloader proposal to avoid build dependency on it
+    allow(subject).to receive(:cpu_mitigations_proposal)
+  end
+
   describe "#initialize" do
     it "instantiates a new proposal settings" do
       expect(Y2Firewall::ProposalSettings).to receive(:instance)
 
-      client
+      described_class.new
```

Comment:
Would be just calling `subject` here enough?

Reply:
no, as subject is already initialized in before, so expect never happen. I will add comment.

Reply:
Ah, I see...